src/parser/transform/expression/transform_function.cpp

```diff
@@ -111,15 +111,15 @@ unique_ptr<ParsedExpression> Transformer::TransformFuncCall(duckdb_libpgquery::P
 
 	auto lowercase_name = StringUtil::Lower(function_name);
 
-	if (root->agg_order) {
-		throw ParserException("ORDER BY is not implemented for aggregates");
-	}
-
 	if (root->over) {
 		if (root->agg_distinct) {
 			throw ParserException("DISTINCT is not implemented for window functions!");
 		}
 
+		if (root->agg_order) {
+			throw ParserException("ORDER BY is not implemented for window functions!");
```

Comment:
Can we add a test that covers this?

Reply:
Yeah, its kind of an odd duck(!) though because there is already an `ORDER BY` clause in the `OVER` . We should come back later and look into how this is supposed to work/merge.